drivers/hud_apr/spec/models/fy2023/datalab_apr/psh_coc_1.rb

```diff
@@ -30,17 +30,16 @@
       )
     end
 
-    # AAQ submitted, 99 in SSNDQ should put client in missing
-    xit 'Q6a' do
+    it 'Q6a' do
       compare_results(
         file_path: result_file_prefix + 'apr/psh_coc_1',
         question: 'Q6a',
-        detail_columns: [
-          :last_name,
-          :first_name,
-          :ssn_quality,
-          :ssn,
-        ],
+        # detail_columns: [
```

Comment:
If you add detail_columns, and a test fails, it will spit out those columns for the failing test. Since I've been doing most of the discovery in-app, they aren't really necessary anymore.  I'll remove.